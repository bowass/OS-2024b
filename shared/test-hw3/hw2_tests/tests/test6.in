create_list
insert_value 20
insert_value 8
insert_value 22
insert_value 23
insert_value 10
insert_value 25
join
insert_value 1
insert_value 9
insert_value 2
insert_value 5
insert_value 6
count_greater 19
insert_value 11
insert_value 3
insert_value 19
join
delete_list
join
exit
