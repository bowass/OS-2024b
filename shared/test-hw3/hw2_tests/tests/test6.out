4 items were counted
